src/coverage.ts

```diff
@@ -57,20 +57,26 @@ export class Coverage {
 
     public findReports(): Promise<string[]> {
         const files = [];
-        const actions = this.vscode.getWorkspaceFolders().map((workspaceFolder) => {
-            return this.globFind(workspaceFolder, "coverage/**/index.html");
-        });
+        let actions: Array<Promise<string[]>> = new Array<Promise<string[]>>();
+
+        const workspaceFolders = this.vscode.getWorkspaceFolders();
+        if (workspaceFolders) {
+            actions = workspaceFolders.map((workspaceFolder) => {
+                return this.globFind(workspaceFolder, "coverage/**/index.html");
+            });
+        }
+
         return Promise.all(actions)
             .then((coverageInWorkspaceFolders) => {
                 // Spread first array to properly concat the file arrays from the globFind
-                return [].concat(...coverageInWorkspaceFolders);
+                return new Array().concat(...coverageInWorkspaceFolders);
             });
     }
 
     public load(path: string): Promise<string> {
         return new Promise<string>((resolve, reject) => {
             this.fs.readFile(path, (err, data) => {
-                if (err) { return reject(err); }
+                if (err && Object.keys(err).length !== 0) { return reject(err); }
```

Comment:
This is a funny way for checking errors, but I can't put undefined / null / etc. in the test with strictTypeChecking turned on.  Maybe it's better to leave this as is and then make a compiler exception in the test?

Reply:
I tinkered with this a bit and found ```as any``` to be an ok middle ground, then you can go back to the more community standard ```if(err) reject(err)``` as well. https://github.com/ryanluker/vscode-coverage-gutters/commit/1c47fbad3e0b90d0118f46592b70f3ac8cd03971

Reply:
Thanks @ryanluker.  Merged that patch into this PR.